### DISCUSSION TITLE
reload configuration without remounting

### DISCUSSION THREAD

**Discussion by mholtz16** [Azure/azure-storage-fuse#1005] [Q&A] [answered]
Is there a way to reload a configuration file (to say, change logging levels) without completely unmounting and mounting again?  I tried sending it a `SIGHUP` but that didn't seem to work. (V2)

**Comment by gapra-msft** [comment, marked as answer]
Blobfuse2 should automatically update the logging level when you modify the config file provided to it. If this is not working for you, please feel free to open an issue and we can investigate further.
